test/kubernetes/e2e/features/deployer/suite.go

```diff
@@ -141,6 +141,10 @@ func (s *testingSuite) TestConfigureProxiesFromGatewayParameters() {
 	s.testInstallation.Assertions.Gomega.Expect(svc.GetAnnotations()).To(
 		gomega.HaveKeyWithValue("svc-anno-key", "svc-anno-val"))
 
+	// check that external traffic policy got passwed through from GatewayParameters to the Service
```

Comment:
s/passwed through/passed/

Reply:
I don't see this value set in the custom GatewayParameters and the upstream default for a Service is `Cluster` instead of `Local`. After asserting the default value, it's important that we also test updating to `Local` in GatewayParameters.